packages/core/database/lib/index.js

```diff
@@ -48,9 +48,7 @@ class Database {
   }
 
   getConnection(tableName) {
-    const schema = this.connection.getSchemaName();
-    const connection = tableName ? this.connection(tableName) : this.connection;
-    return schema ? connection.withSchema(schema) : connection;
+    return tableName ? this.connection(tableName) : this.connection;
```

Comment:
Based on what we discussed, I think we should not change the code here and instead use `connection` instead of `getConnection`. WDYT?